apps/mappings/signals.py

```
@@ -16,9 +16,9 @@
 from apps.mappings.schedules import new_schedule_or_delete_fyle_import_tasks
 from apps.workspaces.models import XeroCredentials, FyleCredential
 from apps.mappings.constants import SYNC_METHODS
-from fyle_integrations_imports.models import ImportLog
+# from fyle_integrations_imports.models import ImportLog
 from apps.xero.utils import XeroConnector
```

Ruff (0.8.2)

`apps.xero.utils.XeroConnector` imported but unused

Remove unused import: `apps.xero.utils.XeroConnector`

(F401)